CHANGELOG.rst

```diff
@@ -40,6 +46,7 @@ Version 1.5.0
 * Modbus read retries works only when empty/no message is received
 * Change test runner from nosetest to pytest
 * Fix Misc examples
+>>>>>>> master
```

Comment:
this is a left-over git conflict marker I think.

Reply:
Yes, this has been taken care already